.changeset/light-hairs-draw.md

```diff
@@ -0,0 +1,6 @@
+---
+"@heroui/listbox": minor
+"@heroui/select": minor
```

Comment:
should be `patch`. we only set react one as minor to avoid double minor bump

Reply:
I see. `value` wasn't actually being used for anything, so `patch` sounds appropriate.